apps/rush-lib/src/cli/actions/GenerateAction.ts

```diff
@@ -105,20 +108,27 @@ export default class GenerateAction extends BaseRushAction {
       // Do an incremental install
       installManager.installCommonModules(InstallType.Normal);
 
-      console.log(os.EOL + colors.bold('(Skipping "npm shrinkwrap")') + os.EOL);
+      if (this.rushConfiguration.packageManager === 'npm') {
+        console.log(os.EOL + colors.bold('(Skipping "npm shrinkwrap")') + os.EOL);
+      } else if (this.rushConfiguration.packageManager === 'pnpm') {
+        console.log(os.EOL + colors.bold('(Deleting "shrinkwrap.yaml")') + os.EOL);
```

Comment:
>(Deleting "shrinkwrap.yaml") [](start = 42, length = 28)

If we're doing "rush generate", then it's fine if PNPM automatically creates the shrinkwrap file for us.

If we're doing "rush install', then the shrinkwrap file should have already been provided by Rush.